invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -4,61 +4,291 @@
+            "type": "string"
+          },
+          "previewer": {
+            "description": "Identifier for previewer needed to preview this file.",
+            "type": "string"
+          },
+          "size": {
+            "description": "Size of the file in bytes.",
+            "type": "integer"
+          },
+          "type": {
+            "description": "File type extension.",
+            "type": "string"
+          },
+          "version_id": {
+            "description": "UUID of the ObjectVersion object.",
+            "type": "string"
+          }
+        },
+        "required": [
+          "bucket",
+          "checksum",
+          "file_id",
+          "key",
+          "size",
+          "type",
+          "version_id"
+        ],
+        "title": "File schema.",
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "_default_preview": {
+      "description": "Default preview for the record.",
+      "type": "string"
+    },
+    "_internal_notes": {
+      "type": "array",
+      "minItems": 0,
+      "uniqueItems": true,
+      "items":{
           "type": "object",
           "properties": {
-              "description": {
-                "description": "Description/abstract for record.",
-                "type": "string"
-              },
-              "description_type": {
-                "description": "Type of description.",
-                "type": "string"
-              },
-              "lang": {
-                "description": "Language of the description. ISO 639-3 language code.",
+              "user": {"type": "string"},
+              "note": {"type": "string"},
+              "timestamp": {
+                "description": "ISO8601 formatted date time stamp.",
                 "type": "string",
-                "maxLength": 3
+                "format": "date-time"
               }
           },
-          "required": ["description", "description_type"]
+          "required": ["user", "note", "timestamp"]
+      }
+    },
+    "_oai": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "OAI-PMH specific information.",
+      "properties": {
+        "id": {
+          "description": "OAI-PMH record identifier.",
+          "type": "string"
+        },
+        "sets": {
+          "description": "OAI-PMH sets for record.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "updated": {
+          "description": "Last OAI-PMH set update (ISO8601 formatted timestamp)",
+          "type": "string"
+        }
+      }
+    },
+    "embargo_date": {
+      "description": "Embargo date of record (ISO8601 formatted date).",
+      "type": "string",
+      "format": "date-time"
+    },
+    "contact": {
+      "type": "string"
+    },
+    "community": {
+      "type": "object",
+      "properties": {
+        "primary": {"type": "string"},
```

Comment:
Since the work on communities will come from Zenodo, I shelved this (https://github.com/inveniosoftware/invenio-rdm-records/issues/50) we can come back once it is integrated into RDM.